README.md

```diff
@@ -72,4 +72,6 @@ const {Session, Cloud} = require('scratchcloud');
 ## Other Information
 You can use the `cloud.get(variableName)` to get the value of a cloud variable by name.
 
-Also, note that you can use the cloud symbol in your variable names, but you do not have to.
+Use `cloud.vars` to get all the names and values of all the cloud variables in a project.
```

Comment:
Maybe mention that it returns an object with the variable names with the cloud symbol as keys, and the values of these variables as the values?